### DISCUSSION TITLE
How many variations are there per avatar style?

### DISCUSSION THREAD

**Discussion by rizkianakbar** [dicebear/dicebear#268] [Q&A] [answered]
May I know how many variations per avatar style? because I plan to use it as the user's default picture and don't want the image to be the same as the others.

**Comment by FlorianKoerner** [comment, marked as answer]
Hi @rizkianakbar 

I have created a help page using the table from discussion #189:

https://dicebear.com/guides/how-many-unique-avatars

**Reply by rizkianakbar** [reply]
thanks!
